winml/lib/Api/LearningModelSessionOptions.cpp

```diff
@@ -52,4 +52,14 @@ STDMETHODIMP LearningModelSessionOptions::SetIntraOpThreadSpinning(boolean allow
   telemetry_helper.SetIntraOpThreadSpinning(allowSpinning != 0);
   return S_OK;
 }
+
+const std::vector<winrt::hstring>& LearningModelSessionOptions::GetCustomOpLibraryPaths() noexcept {
```

Comment:
can we return a span of `const winrt::hstring` instead here?

Reply:
doone